src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -2214,36 +2248,11 @@ define(function(require) {
 						}, {})
 						.value(),
 					name: nameAndNumbersData.portRequestName,
-					bill: _.omitBy({
-						carrier: ownershipConfirmationData.accountOwnership.carrier,
-						name: ownershipConfirmationData.accountOwnership.billName,
-						street_pre_dir: ownershipConfirmationData.serviceAddress.streetPreDir,
-						street_number: ownershipConfirmationData.serviceAddress.streetNumber,
-						street_address: ownershipConfirmationData.serviceAddress.streetName,
-						street_type: ownershipConfirmationData.serviceAddress.streetType,
-						street_post_dir: ownershipConfirmationData.serviceAddress.streetPostDir,
-						extended_address: ownershipConfirmationData.serviceAddress.addressLine2,
-						locality: ownershipConfirmationData.serviceAddress.locality,
-						region: ownershipConfirmationData.serviceAddress.region,
-						postal_code: ownershipConfirmationData.serviceAddress.postalCode,
-						account_number: ownershipConfirmationData.accountInfo.accountNumber,
-						pin: ownershipConfirmationData.accountInfo.pin,
-						btn: ownershipConfirmationData.accountInfo.btn
-					}, _.isEmpty),
 					extra: {
 						numbers_count: _.size(nameAndNumbersData.numbersToPort.formattedNumbers)
-					},
-					transfer_date: monster.util.dateToGregorian(dateAndNotificationsData.targetDate)
-				}, _.isEmpty(notificationEmails) ? {} : {
-					notifications: {
-						email: {
-							send_to: _.size(notificationEmails) === 1
-								? _.head(notificationEmails)
-								: notificationEmails
-						}
 					}
-				});
-
+				}, billSection, transferDateSection, notificationsSection);
+			debugger;
```

Comment:
```suggestion
```

Reply:
:scream:

Reply:
Just found that I left it in the penultimate commit, but then removed it on the last one: 
https://github.com/2600hz/monster-ui/pull/569/commits/2b48b9ea6e5d5fcc98b8a150d6faec6445a45d53

Reply:
Gotcha, I was reviewing commit per commit for context and must have missed that.

P.S.: penultimate means one before last